.maestro/release_tests/password-autofill.yaml

```diff
@@ -45,12 +45,8 @@ tags:
 - tapOn: "Close"
 
 # Validate standard form
-- runFlow:
```

Comment:
These delays were causing some random timeouts in the test so I removed few of them and ensure that the test still pass.